### PR TITLE
Fix typo and add example to message() and highlight()

### PR DESCRIPTION
**Pull request by srawlins**
Typo is s/ANSII/ANSI/ and I found the doc insufficient in describing what format the ANSI code should be supplied in.

### CODE REVIEW

File: lib/src/span.dart [hunk 1]
```diff
@@ -48,10 +48,11 @@ abstract class SourceSpan implements Comparable<SourceSpan> {
   /// Formats [message] in a human-friendly way associated with this span.
   ///
   /// [color] may either be a [String], a [bool], or `null`. If it's a string,
-  /// it indicates an ANSII terminal color escape that should be used to
-  /// highlight the span's text. If it's `true`, it indicates that the text
-  /// should be highlighted using the default color. If it's `false` or `null`,
-  /// it indicates that the text shouldn't be highlighted.
+  /// it indicates an ANSI terminal color escape that should be used to
```
Comment (nex3) [inline lib/src/span.dart]: It's probably a good idea to link "ANSI terminal color escape" to https://en.wikipedia.org/wiki/ANSI_escape_code#Colors.

File: lib/src/span.dart [hunk 2]
```diff
@@ -48,10 +48,11 @@ abstract class SourceSpan implements Comparable<SourceSpan> {
...
+  /// highlight the span's text (e.g. `"\u001b[31m"` will color red). If it's
```
Comment (nex3) [inline lib/src/span.dart]: Nit: [avoid "e.g."](https://www.dartlang.org/guides/language/effective-dart/documentation#avoid-abbreviations-and-acronyms-unless-they-are-obvious).

File: lib/src/span.dart (hunk 1 above)
Comment (srawlins) [reply lib/src/span.dart]: Done.

File: lib/src/span.dart (hunk 2 above)
Comment (srawlins) [reply lib/src/span.dart]: Done.

**Pull request merged**